src/GitHub.TeamFoundation.14/Home/GitHubHomeSection.cs

```diff
@@ -93,12 +76,24 @@ bool IsGitToolsMessageVisible()
                 RepoName = ActiveRepoName;
                 RepoUrl = ActiveRepoUri.ToString();
                 Icon = GetIcon(false, true, false);
+               
+                var weekElapsed = settings.WelcomeMessageLastSeen.GetIso8601WeekOfYear() != DateTimeOffset.Now.GetIso8601WeekOfYear();
```

Comment:
If you save on a sunday and then run this on a monday (assuming sunday is the first day of the week), it'll pop up on both days (also friday/monday, etc). You probably want to just check that x days have elapsed since the last time the message was seen instead?